cgi-bin/dashboard-aii.cgi

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/perl -T
+# ${license-info}
+# ${developer-info
+# ${author-info}
+# ${build-info}
+
+use Config::Simple;
+use JSON::XS;
+use Switch;
+use CGI;
+
+# Configuration file to boot from HD. Following pxelinux convention
+# it must be called default so if the <IpAddressInHex> link
+# is missing the node will boot from the hard disk
+my $boothd = "default";
+
+# Quattor profiles
+my $web_root = "/var/www/html";
+my $profiles_path = "profiles";
```

Comment:
I don't find where this is or should be declared, in aii.conf (it uses `cdb_url` to fetch profiles) ?
Adding fields in `aii.conf` ?

If I use the `cdb_url` parameter, I have to switch to an 'over network' version of this code (read files locally vs fetch via url).